tests/autoyast/verify_cloned_profile.pm

```diff
@@ -0,0 +1,38 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+
+# Summary: Validate if generated autoyast profile corresponds to the expected one
+# Maintainer: Joaquín Rivera <jeriveramoya@suse.com>
+
+use strict;
+use warnings;
+use base 'basetest';
+use testapi;
+use xml_utils;
+use scheduler;
+use autoyast 'init_autoyast_profile';
+
+sub run {
+    my $errors;
+    my $test_data      = get_test_suite_data();
+    my $cloned_profile = $test_data->{cloned_profile};        # get test data related with ay profile
+    my $xpc            = get_xpc(init_autoyast_profile());    # get XPathContext
```

Comment:
$xpc is not descriptive. i guess neither get_xpc but as you dont introduce this, it can be kept as it is. but the variable would improve a lot the readability

Reply:
It is enough descriptive because it looks very much like some copy/paste name convention from examples of use of a library, so it is easy too google it: https://metacpan.org/pod/XML::LibXML::XPathContext. Besides there is a comment already there just in case.

Reply:
i dont get that. yes the comment is here and helps but the variable is used elsewhere. and using a slightly more descriptive name would take away the need of comments. obviously not a problem if you dont want to change it as we are not restrict in those kind of things. thanks for the respond

Reply:
my point is that when I see in the code something that is not meaningful like xpc, that is kind of the acronysm, I always think that is the standard way to name the variable for the use of a library (I have seen in other languages same thing), so in fact it helps when you google for more info to use it. I would not even need the comments, but you're right, this is an opinion.